package.json

```diff
@@ -26,6 +26,7 @@
     "README.md",
     "assets",
     "dist",
+    ".env.defaults",
```

Comment:
Without this, `npx @curveball/a12n-server` doesn't have a source file to copy from (given that there isn't already a .env file)

Reply:
Thanks! wild how broken this was =(